GTG/gtk/data/context_menus.ui

```diff
@@ -28,17 +28,17 @@
   <menu id="task_menu">
     <section>
       <item>
-        <attribute name="label" translatable="yes">Add a subtask</attribute>
+        <attribute name="label" translatable="yes">Add a Subtask...</attribute>
```

Comment:
Unicode typography for ellipsis - https://developer.gnome.org/hig/stable/typography.html.en